app/controllers/cart_products_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CartProductsController < ApplicationController
+  def create
+    products_in_cart = current_user.cart.products
+
+    if products_in_cart.map(&:id).include?(params[:product_id].to_i)
+      product_in_cart = products_in_cart.where(id: params[:product_id]).first
+
+      cart_product = CartProduct.find_by(product_id: product_in_cart, cart_id: current_user.cart.id)
```

Comment:
Once we have the product, we can look up the join model, `CartProduct` and update its quantity.